services/port_manager/src/main/java/com/futurewei/alcor/portmanager/repo/PortRepository.java

```diff
@@ -263,6 +264,33 @@ private String getNeighborCacheName(String suffix) {
         return NEIGHBOR_CACHE_NAME_PREFIX + suffix;
     }
 
+    private List<SubnetPortIds> getSubnetPortIds(List<PortEntity> portEntities) {
+        Map<String, SubnetPortIds> subnetPortIdsMap = new HashMap<>();
+        for (PortEntity portEntity: portEntities) {
+            if ("network:router_interface".equals(portEntity.getDeviceOwner())) {
```

Comment:
A constant string would be better.

Reply:
Fixed, another PR: https://github.com/futurewei-cloud/alcor/pull/519